changelogs/fragments/5067.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - aix_filesystem - Fix example tasks. Previously, the examples in the docs were erroneously using ``community.general.filesystem`` as an argument instead of ``filesystem``. (https://github.com/ansible-collections/community.general/pull/5067)
```

Comment:
Docs-only changes usually have no changelog fragment, so it's better to remove this one.